internal/testutils/testutils_helpers/terraform_plan_helpers.go

```diff
@@ -0,0 +1,151 @@
+package testutils_helpers
+
+import (
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"github.com/pingidentity/pingctl/internal/connector"
+	"github.com/pingidentity/pingctl/internal/connector/pingone/sso"
+)
+
+// Helper function to run terraform plan --generate-config-out on a single resource
+func TestSingleResourceTerraformPlanGenerateConfigOut(t *testing.T, resource connector.ExportableResource, sdkClientInfo *connector.SDKClientInfo) {
+	t.Helper()
+
+	// Get an instance of the PingOne SSO Connector
```

Comment:
I might be misunderstanding, but wouldn't this need to be able to use the other connectors, not just SSO?

Reply:
It was resolved by using `WriteFiles()` directly.